### PR TITLE
Create abhinav.json

### PR DESCRIPTION
**Pull request by abhinavshyju**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://portfolio-6ns.pages.dev/

### CODE REVIEW

**Comment by abhinavshyju** [comment]
The website is currently being built. Approve this domain for a portfolio website, please.

**Comment by abhinavshyju** [comment]
@tonetheman

File: domains/abhinav.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "abhinavshyju"
+    },
+    "record": {
+        "CNAME": "portfolio-6ns.pages.dev"
```
Comment (Stef-00012) [inline domains/abhinav.json:6]: this seems like a default page for a astro app template

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/abhinav.json (hunk 1 above)
Comment (abhinavshyju) [reply domains/abhinav.json:6]: <html>
<body>

Error: Failed to publish your function. Got error: Uncaught ReferenceError: MessageChannel is not defined   at chunks/_@astro-renderers_4GnOaUIv.mjs:6534:16 in requireReactDomServer_browser_production   at chunks/_@astro-renderers_4GnOaUIv.mjs:12531:8 in requireServer_browser   at chunks/_@astro-renderers_4GnOaUIv.mjs:12543:29
--



</body>
</html> Im stuck on this error. i will fix it asap

**Pull request closed without merging**